### PR TITLE
Add PHP 7.4

### PR DESCRIPTION
**Pull request by Mezzle**

### CODE REVIEW

**Comment by Neodork** [comment]
- [X] I have formatted my code with the [PSR-2](http://www.php-fig.org/psr/psr-2/) coding style before submitting my PR.
- [X] I have created an issue which accompanies my PR: https://github.com/weprovide/valet-plus/issues/427.

**I have read the contribution guidelines and am targeting the branch `2.x`:**  
Because this is a Feature which is Backwards Compatible.  

**Changelog entry:**  
Short description of your work as explained by: [Keep A Changelog](https://keepachangelog.com).

- Added support for switching to PHP 7.4

**Comment by Vinai** [comment]
Can you possible provide an ETA when 7.4 will be available?

**Comment by Neodork** [comment]
@Vinai I hope to have the last bugs ironed out by the end of next week. No hard deadlines though.

File: azure-pipelines.yml [hunk 1]
```diff
@@ -127,7 +127,7 @@ steps:
 
 - script: |
     ./valet use 7.4
-    php -v
+    php -v | grep 7.4I
```
Comment (Neodork) [inline azure-pipelines.yml]: @Mezzle This is a typo I assume?

**Comment by Neodork** [comment]
@Mezzle Good news, pipelines are moving forward after fixing the formula and pre-build binaries url paths. 

It's now failing on Ioncube for 7.4 because there is no .so file for ioncube on 7.4 yet. Can you remove the test for ioncube for PHP 7.4? We'll probably have to open a new issue to solve this, it should throw an warning and fail silently.

**Comment by Mezzle** [comment]
Even better, I wrote code to handle it failing.

**Comment by Mezzle** [comment]
New problem: `mysql-utilities` has been removed from homebrew

https://github.com/Homebrew/homebrew-core/commit/409722c430b8a24596933174a17ca49e6effb783#diff-313a6a3d25aa041ee3dc3cbd65d4f22b

**Comment by Mezzle** [comment]
As far as I can tell, that package isn't used though?

**Comment by Vinai** [comment]
Yay, all green! :)

**Comment by Neodork** [comment]
@Vinai It should be available in the latest `2.x` branch ;).

**Comment by Vinai** [comment]
Will you be rolling it into the tagged releases, too?

**Pull request merged**